convokit/model/corpusHelper.py

```diff
@@ -284,15 +288,18 @@ def merge_utterance_lines(utt_dict):
     return new_utterances
 
 
-def initialize_conversations(corpus, utt_dict, convos_data):
+def initialize_conversations(corpus, utt_dict, convos_data, convo_to_utts=None):
```

Comment:
Needs some docs indicating the cases where `convo_to_utts` is passed. Type annotations / docs would be helpful too.

Reply:
I've updated the documentation, but as far as type annotations I noticed that there are none throughout this file and assumed this was by design (I'm trying to stay consistent with the rest of the file)

Reply:
Less by design, more like lack of time :P